test/mocks/MockScratchConnector.scala

```diff
@@ -17,24 +17,20 @@
 package mocks
 
 import scala.concurrent.{ExecutionContext, Future}
-
 import org.scalamock.handlers.CallHandler
 import org.scalamock.scalatest.MockFactory
-
 import play.api.libs.json.JsValue
-
 import uk.gov.hmrc.http.HeaderCarrier
+import connectors.ScratchConnector
+import models.{RequestOutcome, ScratchResponse}
 
-import connectors.GuidanceConnector
-import models.{RequestOutcome, SaveScratchSubmissionResponse}
-
-trait MockGuidanceConnector extends MockFactory {
+trait MockScratchConnector extends MockFactory {
 
-  val mockGuidanceConnector: GuidanceConnector = mock[GuidanceConnector]
+  val mockGuidanceConnector: ScratchConnector = mock[ScratchConnector]
```

Comment:
I would change the val and object named mockGuidanceConnector to mockScratchConnector.

Reply:
Updated in latest commit